platform/core/src/services/HangingProtocolService/lib/validator.js

```diff
@@ -1,69 +1,231 @@
 import validate from 'validate.js';
 
-validate.validators.equals = function(value, options, key, attributes) {
-  const testValue = options?.value ?? options;
-  if (value !== testValue) {
-    return key + 'must equal ' + testValue;
+/**
+ * check if the value is strictly equal to options
+ *
+ * @example
+ * value = ['abc', 'def', 'GHI']
```

Comment:
Does ['abc'] match 'abc' and vice-versa?  That one is probably a use case worth testing because in the DICOM world they are always arrays.